datasafe-business/src/test/java/de/adorsys/datasafe/business/impl/e2e/BasicFunctionalityWithConcurrencyTest.java

```diff
@@ -153,11 +155,15 @@ void writeToPrivateListPrivateInDifferentThreads(WithStorageProvider.StorageDesc
         metricCollector.setStorageType(storage.getClass().getSimpleName());
         metricCollector.setNumberOfThreads(poolSize);
         metricCollector.writeToJSON();//json files in target folder
+
+        deleteTestFile(testFile);
```

Comment:
why. Test creates with generateTestFile and eventually deletes with deleteTestfile. Otherwise generateTestFile should be done in BeforeEach too. But I did not want to change too much.